src/NuGetUtility/Wrapper/NuGetWrapper/Protocol/WrappedPackageMetadata.cs

```diff
@@ -28,6 +28,10 @@ public WrappedPackageMetadata(ManifestMetadata metadata)
 
         public string Summary => _metadata.Summary;
 
+        public string Copyright => _metadata.Copyright;
+
+        public string Authors => string.Join(',', _metadata.Authors); // https://learn.microsoft.com/en-us/nuget/reference/nuspec#authors
```

Comment:
Can you please keep the authors as array - it will be much easier to parse the output by a machine if we concatinate with comma.